ppml/trusted-big-data-ml/scala/docker-occlum/run_spark_on_occlum_glibc.sh

```diff
@@ -386,6 +437,19 @@ id=$([ -f "$pid" ] && echo $(wc -l < "$pid") || echo "0")
 arg=$1
 case "$arg" in
     init)
+       export RUNTIME_ENV="native"
```

Comment:
Why do we need to separate driver and executor init?

Reply:
for next steps, now each executor have to register again.